manageiq-operator/api/v1alpha1/helpers/miq-components/generators.go

```diff
@@ -20,6 +21,13 @@ func generateEncryptionKey() string {
 }
 
 func generatePassword() string {
-	buf := randomBytes(8)
-	return hex.EncodeToString(buf)
+	for {
+		buf := randomBytes(8)
+		password := hex.EncodeToString(buf)
+		if match, err := regexp.MatchString(`\D+`, password); err == nil && match {
```

Comment:
Will it return a blank password?  I was expecting it would try again in a loop?  A blank password might be worse if nothing is checking it.

Reply:
Oh nevermind, I didn't see the `for` and forgot that's a free loop construct in go

Reply:
I don't think it can be blank, it's 8 random bytes, so 16 characters.